### ISSUE TITLE
Same issue on R9 380 with Rocket Lake.

### ISSUE THREAD

**Issue by lowercase00** [PracticalPear413/RyzenTosh_1600X-RX580_2048SP#1]
If I don't spoof, system will crash at any sign of graphics in the screen. If I spoof, no hardware acceleration. Also tried the SSDT fix, but nothing. Did you end up fixing it?

**Comment by PracticalPear413** [comment]
> If I don't spoof, system will crash at any sign of graphics in the screen. If I spoof, no hardware acceleration. Also tried the SSDT fix, but nothing. Did you end up fixing it?

Nope. My system didn't crash without spoof, it just never had hardware acceleration.
If your system is crashing, without any kind of spoof, that means something else is wrong because at least the CPU should be outputting the display.

**Comment by lowercase00** [comment]
Actually not in my case, since I'm on the 11th Generation Rocket Lake, so no iGPU enabled... I'm guessing this is indeed related to the GPU.
